src/openrct2/rct1/Tables.cpp

```diff
@@ -68,52 +68,52 @@ namespace RCT1
         return map[colour];
     }
 
-    PeepSpriteType GetPeepSpriteType(uint8_t rct1SpriteType)
+    ::PeepSpriteType GetPeepSpriteType(PeepSpriteType rct1SpriteType)
```

Comment:
Why are all these `::PeepSpriteType`? It's defined in the `RCT1` namespace, which we are in.

Reply:
This is a conversion function from rct1 to openrct2. As the openrct2 fields are not in a namespace yet you get this horrible stuff